app/views/api/v2/settings/main.json.rabl

```diff
@@ -3,3 +3,7 @@ object @setting
 extends "api/v2/settings/base"
 
 attributes :value, :description, :category, :settings_type, :default, :created_at, :updated_at
+
+node do |s|
+  { :category_name => s.class.humanized_category }
```

Comment:
1. This will return `nil` for most categories, it should follow the same logic as the UI helper (`s.class.name.demodulize`) when humanized_category is nil.
1. Missing `_( .. )` of the result.